### ISSUE TITLE
Nuget Cryptography dependency

### ISSUE THREAD

**Issue by deansilkyocean** [imgix/Imgix-CSharp#12]
Hi,
When I try and install via nuget in VS2017 Community I get an error about Cryptography dependency.  Am I missing something?

```
Attempting to gather dependency information for package 'Imgix-CSharp.1.0.1' with respect to project 'cightedfrontendwebforms', targeting '.NETFramework,Version=v4.6.1'
Gathering dependency information took 8.36 sec
Attempting to resolve dependencies for package 'Imgix-CSharp.1.0.1' with DependencyBehavior 'Lowest'
Install-Package : Unable to resolve dependency 'Cryptography'. Source(s) used: 'nuget.org', 'Microsoft Visual Studio Offline Packages'.
At line:1 char:1
+ Install-Package Imgix-CSharp
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

Thanks
Dean

**Comment by enghch** [comment]
I am seeing the same issue. Just wanted to upvote it for exposure.

**Comment by hashknot** [comment]
Hi @deansilkyocean, @enghch

I published v1.0.2 which removes Cryptography dependency. Can you guys confirm if it's working now?

Thanks!
